### PR TITLE
feat(organization): backfill Asset search field for owner label TASK-1334

### PR DESCRIPTION
**Pull request by noliveleger**
### 📣 Summary
Populated the asset search field to include the owner username and organization name for improved search functionality.

### 📖 Description
The asset search field has been updated to include tthe owner username and organization name, ensuring better usability when searching for asset owner labels. This backfill operation populates the field for existing assets where the owner label was previously missing.

### CODE REVIEW

File: kobo/apps/long_running_migrations/jobs/0004_back_fill_asset_search_field_for_owner_label.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Generated on 2024-12-19 15:34
+from django.db.models import Prefetch
+from more_itertools import chunked
+
+from kobo.apps.organizations.models import Organization
+from kpi.models.asset import Asset
+
+
+def run():
```
Comment (Guitlle) [inline kobo/apps/long_running_migrations/jobs/0004_back_fill_asset_search_field_for_owner_label.py:9]: Should we have unit tests for at least some of these migrations? Though, I would expect them to get tested when devs pull from main and the local environment executes them from time to time. But maybe just for QA purposes, either having unit tests or reproducible previews.

**Review by Guitlle — APPROVED** [review]
It looks good, though I did not test the migration scripts on my local env

**Review by Guitlle — APPROVED** [review]
Looks good, but I didn't test the long running migration

File: kobo/apps/long_running_migrations/jobs/0004_back_fill_asset_search_field_for_owner_label.py (hunk 1 above)
Comment (noliveleger) [reply kobo/apps/long_running_migrations/jobs/0004_back_fill_asset_search_field_for_owner_label.py:9]: We decided to do simple tests (e.g.: validate the data is correctly added) in another PR.

**Pull request merged**